tests/std/tests/VSO_0102478_moving_allocators/test.cpp

```diff
@@ -76,6 +76,7 @@ using expected_table_t = vector<test_info>;
 #pragma warning(disable : 4640) // 'variable': construction of local static object is not thread-safe
 #if _ITERATOR_DEBUG_LEVEL == 0
 
+#if defined(MSVC_INTERNAL_TESTING) || defined(__EDG__) // TRANSITION, 16.10p1

```

Comment:
Agreed -- then, it'll only be activated for Clang in the runtime calls (since `__cpp_lib_constexpr_string` isn't defined for Clang yet).  Good catch!